spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/service/PackageMetadataService.java

```diff
@@ -188,7 +188,7 @@ private boolean checkIfPackageIsFromLocalRepo(List<String> errorMessages, Packag
 				try {
 					if (!packageRepository.isLocal()) {
 						Resource resource = resourceLoader.getResource(packageRepository.getUrl()
-								+ File.separator + "index.yml");
+								+ "/index.yml");
```

Comment:
Isn't the `file.separator` working in your case? Could you elaborate on your changes?

Reply:
Do not use a local (OS specific) value of File.separator when you build URL to a remote service.

Reply:
File.separator isn't URL separator in common case. For example, windows File.separator is '\'. Result in this case will be <repository URL>\index.yml. This isn't valid URL. And you get error.

Reply:
@iTytar Now I see the issue. Thanks for the clarification. Will merge this PR after excluding the application.yml change.